backend/api/Services/IsarService.cs

```diff
@@ -317,11 +317,13 @@ public async Task<MediaConfig> GetMediaStreamConfig(Robot robot)
                 logger.LogError("{Message}: {ErrorResponse}", message, errorResponse);
                 throw new ConfigException(message);
             }
-            if (response.Content is null)
```

Comment:
Is this equivalent to "if (response.StatusCode == HttpStatusCode.NoContent)"?

Reply:
The never one looks better, just wondering if it covers the same bases

Reply:
No, this line doesn't seem to trigger regardless of content